hazelcast/src/test/java/com/hazelcast/internal/ascii/RestCPSubsystemTest.java

```diff
@@ -71,7 +71,7 @@
 
     private Config config = new Config();
     private String clusterName = config.getClusterName();
-    private String groupPassword = config.getClusterPassword();
+    private String groupPassword = null;
```

Comment:
Shall we remove it completely?

Reply:
It would make sense to leave it here if there is an EE variant with the security enabled, but I don't see such test class in the EE, so I'll drop this one.